README.md

```diff
@@ -60,15 +61,15 @@ Example Playbook
 - name: Converge
   hosts: all
   roles:
-    - role: lean_delivery.docker
+    - role: ansible-role-docker
```

Comment:
Why changing the name?

Reply:
Because this format `lean_delivery.docker` is used in `requirements.yml` only.